scripts/partition_manager.py

```diff
@@ -796,6 +812,28 @@ def test():
     offset = get_required_offset(align={'end': 800}, start=0, size=1000, move_up=False)
     assert offset == 600
 
+    failed = False
+    try:
+        offset = get_required_offset(align={'end': ["CONFIG_VAR"]}, start=0, size=1000, move_up=False)
+    except TypeError:
+        assert True
+    else:
+        assert False
```

Comment:
This code basically only verifies that this line:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/b1301e153adf7abaf6a2e8564df0954d1c427cac/scripts/partition_manager.py#L459
will raise a `TypeError`.

As far as I can tell `test()` never verifies that https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/b1301e153adf7abaf6a2e8564df0954d1c427cac/scripts/partition_manager.py#L375-L389 raises the error when value is wrong.

Reply:
Doesn't line 823-836 verify it by calling a function which again calls `align_if_required` ?